cloud/blockstore/libs/endpoints/endpoint_manager.cpp

```diff
@@ -581,11 +611,11 @@ NProto::TListEndpointsResponse TEndpointManager::DoListEndpoints(
 
     NProto::TListEndpointsResponse response;
     auto& endpoints = *response.MutableEndpoints();
-    endpoints.Reserve(Requests.size());
+    endpoints.Reserve(Endpoints.size());
 
-    for (auto it: Requests) {
+    for (auto it: Endpoints) {
```

Comment:
`it` is not an iterator, but a pair. Maybe `auto [_, ep]`?

Reply:
done